pkg/operator/operator.go

```diff
@@ -259,8 +261,10 @@ func LoadOperatorConf(cmd *cobra.Command) *Conf {
 	c.NS = util.KubeObject(bundle.File_deploy_namespace_yaml).(*corev1.Namespace)
 	c.SA = util.KubeObject(bundle.File_deploy_service_account_yaml).(*corev1.ServiceAccount)
 	c.SAEndpoint = util.KubeObject(bundle.File_deploy_service_account_endpoint_yaml).(*corev1.ServiceAccount)
+	c.SAUI = util.KubeObject(bundle.File_deploy_service_account_ui_yaml).(*corev1.ServiceAccount)
```

Comment:
do we need the service account?

Reply:
required by olm, referenced [here](https://github.com/baum/noobaa-operator/blob/727593403702dcf80d920cea3c921f9dc236ff3d/pkg/olm/olm.go#L248-L252)